Nodejs/Product/Nodejs/Project/NodeModulesNode.cs

```diff
@@ -251,12 +213,11 @@ private void WriteNpmLogToOutputWindow(NpmLogEventArgs args)
         private void NpmController_CommandStarted(object sender, NpmCommandStartedEventArgs e)
         {
             StopNpmIdleTimer();
-            lock (this._commandCountLock)
+            lock (this._commandLock)
             {
-                ++this._npmCommandsExecuting;
+                var message = string.Format(CultureInfo.CurrentCulture, Resources.NpmStatusExecuting, e.CommandText);
```

Comment:
Why don't we need to count the number of invocations anymore?

Reply:
Really counting invocations didn't do anything at all. We just increase and decrease the value but we never really use it for anything.

Reply:
I mean we use to use it I suppose in case we call the status bar more than once. But on this new method it no longer serves any purpose.